go/deploy/builderd/contrib/grafana-dashboards/README.md

```
@@ -0,0 +1,408 @@
+| `builderd-overview.json` | High-level service metrics and health | Operations, SRE |
+| `builderd-builds.json` | Build execution and performance metrics | Developers, Operations |
+| `builderd-tenants.json` | Multi-tenant usage and quota monitoring | Account Management, Operations |
+| `builderd-infrastructure.json` | System resources and infrastructure health | SRE, Infrastructure |
+| `builderd-security.json` | Security events and isolation monitoring | Security, Operations |
+
+## Quick Setup
+
+### Import Dashboards
+
+```bash
+# Import all dashboards using Grafana API
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)